samples/net/wifi/wifi_shell/README.rst

```diff
@@ -1,7 +1,7 @@
-.. _wifi_sample:
+.. _wifi_shell_sample:
```

Comment:
I'm not able comment on the sample's CMakeLists.txt, but you should also update the project name in there.